easybuild/easyblocks/q/quantumespresso.py

```diff
@@ -361,28 +464,19 @@ def sanity_check_step(self):
         if 'yambo' in self.cfg['buildopts']:
             yambo_bins = ["a2y", "p2y", "yambo", "ypp"]
 
-        pref = self.install_subdir
+        d3q_bins = []
+        if 'd3q' in self.cfg['buildopts']:
+            d3q_bins = ['d3_asr3.x', 'd3_import3py.x', 'd3_lw.x', 'd3_q2r.x',
+                        'd3_qq2rr.x', 'd3q.x', 'd3_r2q.x', 'd3_recenter.x',
+                        'd3_sparse.x', 'd3_sqom.x', 'd3_tk.x']
 
         custom_paths = {
-                        'files': [os.path.join(pref, 'bin', x) for x in bins] +
-                                 [os.path.join(pref, 'upftools', x) for x in upftools] +
-                                 [os.path.join(pref, 'WANT', 'bin', x) for x in want_bins] +
-                                 [os.path.join(pref, 'YAMBO', 'bin', x) for x in yambo_bins],
-                        'dirs': [os.path.join(pref, 'include')]
+                        'files': [os.path.join('bin', x) for x in bins] +
+                                 [os.path.join('bin', x) for x in upftools] +
+                                 [os.path.join('bin', x) for x in want_bins] +
+                                 [os.path.join('bin', x) for x in yambo_bins] +
+                                 [os.path.join('bin', x) for x in d3q_bins],
+                        'dirs': ['bin']
                        }
 
         super(EB_QuantumESPRESSO, self).sanity_check_step(custom_paths=custom_paths)
-
-    def make_module_req_guess(self):
-        """Custom path suggestions for Quantum ESPRESSO."""
-        guesses = super(EB_QuantumESPRESSO, self).make_module_req_guess()
-
-        # order matters here, 'bin' should be *last* in this list to ensure it gets prepended to $PATH last,
-        # so it gets preference over the others
-        # this is important since some binaries are available in two places (e.g. dos.x in both bin and WANT/bin)
-        bindirs = ['upftools', 'WANT/bin', 'YAMBO/bin', 'bin']
```

Comment:
@akesandgren This can be removed thanks to the custom `install_step`, but I don't see the `YAMBO/bin` binaries being copied?

I see these for older versions of QE (5.2.1 & older) in `YAMBO/bin`:
```
a2y  p2y  yambo  ypp
```